modules/mobile/lib/scripts/appointments_list_validation.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'json'
+require 'pry'
```

Comment:
Leaving pry here because you'll probably want it when running this. It's sometimes useful to examine the data coming from the backend.